app/src/main/kotlin/com/intive/picover/main/navigation/view/MainScreen.kt

```diff
@@ -34,6 +36,10 @@ fun MainScreen(activity: Activity, snackbarHostState: SnackbarHostState) {
 	val windowSize = calculateWindowSizeClass(activity)
 	val bottomSheetNavigator = rememberBottomSheetNavigator()
 	val navController = rememberNavController(bottomSheetNavigator)
+	DisposableEffect(Unit) {
+		NavControllerHolder.attach(navController)
+		onDispose { NavControllerHolder.detach() }
+	}
```

Comment:
I can only imagine how committing was painful 😄

Reply:
My eyes are still bleeding when I'm looking at this 😄